.storybook/preview.js

```diff
@@ -58,7 +45,8 @@ const customViewports = {
 };
 
 export const parameters = {
-  layout: "fullscreen",
+  /* layout: "fullscreen", not sure what this was for but it's overriding the docs layout */
```

Comment:
**suggestion** (non-blocking): unless there's a very good reason, I'd not bother with this comment, or at least remove it before merging (fair enough if it was left for the benefit of reviewers!). I trust you that you've checked and the display is fine without this, in which case it doesn't matter why it was there before - and anyone can check the git history if they're interested in how it used to be.

Reply:
having now looked at [the docs](https://storybook.js.org/docs/configure/story-layout#global-layout) and inspected both old and new versions, it seems that we set the "fullscreen" layout to remove 16px of padding that Storybook is adding in the Canvas layout. (We're explicitly adding it on the new line below, but it appears this is the default anyway.) I don't know how much it matters but it might mess up some layouts, so unless there's good reason I'd sooner keep this - and it looks like the new way is just to replace `layout: "padded"` with `layout: "fullscreen"` in the line below. If as you say the issue with the "fullscreen" layout was just with the docs then I assume doing that will keep the docs as you have them but put the Canvas back like it was before. (Not had a chance to test it though.)

Reply:
I believe this was again in the very early days where I was just trying to get Storybook to even run and I'm not sure it's ever been revisited as nothing seemed to be broken. I will have a play with what you've suggested though, sounds likely that that's what we need to do here